packages/apps/human-app/server/src/modules/job-assignment/spec/job-assignment.fixtures.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  JobAssignmentCommand,
+  JobAssignmentData,
+  JobAssignmentDto,
+  JobAssignmentResponse,
+  JobsFetchParamsCommand,
+  JobsFetchParamsData,
+  JobsFetchParamsDto,
+  JobsFetchResponse,
+  JobsFetchResponseItem,
+} from '../interfaces/job-assignment.interface';
+import {
+  SortField,
+  SortOrder,
+  StatusEnum,
+} from '../../../common/enums/job-assignment';
+
+export const jobAssignmentDtoFixture: JobAssignmentDto = {
+  exchange_oracle_url: 'test_url',
```

Comment:
test with an example URL (https://www.example.com/api)

Reply:
Fixed in: 051f0adc